docs/src/integrations/iceberg.md

```diff
@@ -412,6 +409,181 @@ sequenceDiagram
+The table's metadata and data remain in lakeFS-managed storage, which are used as the pushed table's location.
+
+**API Endpoint**: `POST /iceberg/remotes/{catalog-id}/push`
+
+**Parameters**:
+- `source`: The lakeFS table location (repository, branch/reference, namespace, table name)
+- `destination`: The remote catalog location (namespace, table name)
+- `force_update`: (optional, default: `false`) Override the table if it already exists in the remote catalog
+- `create_namespace`: (optional, default: `false`) Create the namespace in the remote catalog if it doesn't exist
+
+**Example**:
+
+```bash
+curl -X POST "https://lakefs.example.com/iceberg/remotes/aws_glue_us_east_1/push" \
+  -H "Authorization: Bearer $LAKEFS_ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "source": {
+      "repository_id": "my-repo",
+      "reference_id": "main",
+      "namespace": ["default", "features"],
+      "table": "image_properties"
+    },
+    "destination": {
+      "namespace": ["main_features"],
+      "table": "image_properties"
+    },
+    "force_update": true,
+    "create_namespace": true
+  }'
+```
+
+This example pushes the table `my-repo.main.default.features.image_properties` from lakeFS to the AWS Glue catalog as `main_features.image_properties`,
+creating the namespace if needed and overwriting any existing table.
+
+### Pull from remote
+
+Pull operations update a lakeFS table with changes from a remote catalog.
+This is useful after external tools have modified a table previously pushed from lakeFS.
+
+**API Endpoint**: `POST /iceberg/remotes/{catalog-id}/pull`
+
+**Parameters**:
+- `source`: The remote catalog location (namespace, table name)
+- `destination`: The lakeFS table location (repository, branch/reference, namespace, table name)
+- `force_update`: (optional, default: `false`) Override the table in lakeFS if metadata conflicts exist
+- `create_namespace`: (optional, default: `false`) Create the namespace in lakeFS if it doesn't exist
+
+**Example**:
+
+```bash
+curl -X POST "https://lakefs.example.com/iceberg/remotes/aws_glue_us_east_1/pull" \
+  -H "Authorization: Bearer $LAKEFS_ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "source": {
+      "namespace": ["main_features"],
+      "table": "image_properties"
+    },
+    "destination": {
+      "repository_id": "my-repo",
+      "reference_id": "main",
+      "namespace": ["default", "features"],
+      "table": "image_properties"
+    }
+  }'
+```
+
+This example pulls changes from the Glue table `main_features.image_properties` back into the lakeFS table `my-repo.main.default.features.image_properties`.
+
+### Important Notes
+
+1. **Storage Location**: Pulled tables' `metadata.json` file must reside in a storage location to which lakeFS has read access to.
+
+2. **Atomicity**: Push and pull operations are not atomic. If an operation fails partway through, manual intervention may be required.
```

Comment:
I suggest adding a very basic troubleshooting guide here: should a failed operation be retried? if it needs another manual intervention, what operations should the user perform?

Reply:
Adding these details here will require diving into some confusing technical details about the internals of the operations, from which I tried to avoid.
Added a comment about contacting support - I suggest to update this in case we'll encounter some more issues here.
(At this point, I think we should remove as much friction as possible, to increase the adoption of the catalog.)